spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+require 'spec_helper'
+
+feature 'user signs up' do
+
+  scenario 'specify valid and required information' do
+    visit root_path
```

Comment:
Consider how using a `context` and/or `describe` block might make these feature specs DRY-er.